packages/style-engine/class-wp-style-engine.php

```diff
@@ -423,20 +445,25 @@ public function get_block_supports_styles( $block_styles, $options ) {
 		}
 
 		// Build CSS rules output.
-		$css_selector = isset( $options['selector'] ) ? $options['selector'] : null;
-		$style_rules  = new WP_Style_Engine_CSS_Declarations( $css_declarations );
+		$css_selector        = isset( $options['selector'] ) ? $options['selector'] : null;
+		$should_enqueue      = isset( $options['enqueue'] ) && true === $options['enqueue'];
```

Comment:
We should create another public method, e.g., `wp_style_engine_enqueue_block_supports_styles` to hide this implementation. 

This is just testing for now.